docs/source/create/entities/zone-entity.rst

```diff
@@ -0,0 +1,166 @@
++------------------------+---------------------------------------------------------------------------------------------+
+| Key Light              | This sets a directional light, e.g. a sun, for the entire zone.                             |
++------------------------+---------------------------------------------------------------------------------------------+
+| Skybox                 | This lets you set a so called "Skybox". This is commonly used to apply a sky texture to     |
+|                        | the zone.                                                                                   |
++------------------------+---------------------------------------------------------------------------------------------+
+| Ambient Light          | This is used to apply light coming from the skybox. This is used if you have light coming   |
+|                        | from more than just a sun. For example if you have multiple suns.                           |
++------------------------+---------------------------------------------------------------------------------------------+
+| Haze                   | Creates fog or mist.                                                                        |
++------------------------+---------------------------------------------------------------------------------------------+
+| Bloom                  | This enables a visual effect known as "bloom". A slight bloom effect could be used to make  |
+|                        | zones appear bright.                                                                        |
++------------------------+---------------------------------------------------------------------------------------------+
+| Avatar Priority        | This property changes the update priority of avatar movements by allocating more bandwidth  |
+|                        | which has been reserved by the avatar mixer. It is commonly used on live events to have     |
+|                        | entertainers' avatar movements still be smooth, even with a huge crowd also being updated.  |
++------------------------+---------------------------------------------------------------------------------------------+
+
+^^^^^^^^^
+Key Light
+^^^^^^^^^
+
+Key lighting is used to illuminate a zone. You can think of the key light as a sun which illuminates a zone from a specific direction.
+The following properties are available for key lighting:
+
++------------------------+---------------------------------------------------------------------------------------------+
+| Property               | Description                                                                                 |
++========================+=============================================================================================+
+| Key Light Color        | Sets the color of the key light. This could be used to simulate a sunset.                   |
```

Comment:
Changed to "This could be used to simulate the color of a sunset. " to make it more clear.

Reply:
I was recommending just removing the example entirely and putting the reference as below since I guess that would be a more fitting place to put such an example, so that people know that everything below plays into such a thing?